src/detection.c

```diff
@@ -226,3 +218,39 @@ float get_side_right_distance(void)
 {
 	return distance[SENSOR_SIDE_RIGHT_ID];
 }
+
+/**
+ * @brief Calculate and return the side sensors error.
+ *
+ * Taking into account that the walls are parallel to the robot, this function
+ * returns the distance that the robot is moved from the center parallel line
+ * of the maze.
```

Comment:
"center of the corridor"? instead of "center parallel line of the maze"?

Reply:
good